docs/throttling.md

```diff
@@ -0,0 +1,58 @@
+
+## The 3G network throttling preset
+
+This is the standard recommendation for 3G throttling:
+
+- Latency: 150ms
+- Throughput: 1.6Mbps down / 750 Kbps up.
+- Packet loss: none.
+
+These exact figures are used as the [WebPageTest "Mobile 3G - Fast" preset](https://github.com/WPO-Foundation/webpagetest/blob/master/www/settings/connectivity.ini.sample) and [Lighthouse's throttling default](https://github.com/GoogleChrome/lighthouse/blob/master/lighthouse-core/lib/emulation.js).
+
+## Throttling basics
+
+1. The DevTools network throttling, which Lighthouse uses, is implemented within Chrome at the _request-level_. As a result, it has some downsides that are now summarized in this doc: [Network Throttling & Chrome - status](https://docs.google.com/document/d/1TwWLaLAfnBfbk5_ZzpGXegPapCIfyzT4MWuZgspKUAQ/edit). The TLDR: while it's a [decent approximation](https://docs.google.com/document/d/1uS9SH1KpVH31JAmf-iIZ61VazwAF9MrCVwETshBC4UQ/edit), it's not a sufficient model of a slow connection.
```

Comment:
do you want to mention in the TLDR that's why we have [multipliers](https://github.com/GoogleChrome/lighthouse/blob/master/lighthouse-core/lib/emulation.js#L38) to achieve comparable speeds

Reply:
sure sgtm